From 37a59cd06d5fbfa83eac0d624b5aa5ebc56f4998 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Fri, 21 Oct 2016 01:23:14 +0100
Subject: [PATCH] Butcher soname to enable building of "libpcre3"

This is a very odd Debianism that has existed since the dawn of time.
Obviously, we didn't want to use this anywhere in Solus. *Unfortunately*,
this *NON STANDARD* soname is required by certain steam games. i.e. they
build for a specific Linux distribution. Looking at you, Alien Isolation.

This will allow us to provide our main libpcre package with the a different
soname, and we'll make it impossible to link against the library and make
it devoid of headers, etc. Because frankly this stinks. Same as curl-gnutls.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 configure.ac | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 3cefaf1..4bdf9e8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -17,10 +17,10 @@ m4_define(pcre_date, [2016-06-14])
 # 50 lines of this file. Please update that if the variables above are moved.
 
 # Libtool shared library interface versions (current:revision:age)
-m4_define(libpcre_version, [3:7:2])
-m4_define(libpcre16_version, [2:7:2])
-m4_define(libpcre32_version, [0:7:0])
-m4_define(libpcreposix_version, [0:4:0])
+m4_define(libpcre_version, [16:1:13])
+m4_define(libpcre16_version, [16:1:13])
+m4_define(libpcre32_version, [16:1:13])
+m4_define(libpcreposix_version, [16:1:13])
 m4_define(libpcrecpp_version, [0:1:0])
 
 AC_PREREQ(2.57)
-- 
2.10.1

